docs/sources/whatsnew/whats-new-in-v9-5.md

```diff
@@ -77,6 +77,12 @@ If you want to enable this feature, you'll need to enable the `onlyExternalOrgRo
 
 You can prevent the synchronization of organization roles from the authentication provider. For more information, refer to [skip org role sync]({{< relref "../setup-grafana/configure-grafana/#authgrafana_com-skip_org_role_sync/" >}}).
 
+## Removing the button to create API keys from the UI
+
+With this update we are going one step further in deprecating API Keys. Previously, the migration status display may have suggested that the API Key migration would be a significant issue. To address this, we have revised the text on the API keys page and now allow creating new API keys exclusively via our HTTP API. This change is part of our long-term strategy for sunsetting API keys.
```

Comment:
I think I wouldn't even mention the rewording of the migration. We should also add something about service accounts - currently the user might not be aware that SAs are the way to go 😄 

```suggestion
With this update we are going one step further in deprecating API Keys in favour of [service accounts](<LINK>). We have removed the button to create new API keys through Grafana's user interface, and now only allow creating API keys via our HTTP API. We urge you to migrate your existing API keys to service accounts, and opt for new service accounts instead of new API keys. This change is part of our long-term strategy for sunsetting API keys.
```